frontend/package.json

```diff
@@ -3,15 +3,23 @@
   "version": "1.0.0",
   "description": "civis: An Ionic project",
   "dependencies": {
-    "gulp": "^3.5.6",
-    "gulp-sass": "^1.3.3",
+    "angular-bootstrap-datetimepicker": "^1.0.1",
```

Comment:
A lot of dependencies added here are client side dependencies (angular-*, bootstrap, highcharts*) so they should not be included as part of NPM.